### ISSUE TITLE
importing PathParticle

### ISSUE THREAD

**Issue by edap** [pixijs/pixi-particles#35]
I'm building a pixi videogame using es6 and babel. I've used this boilerplate https://github.com/magnonellie/pixi-boilerplate and I've added `pixi-particles` as dependency.
I'm importing pixi and pixi-particles as following:

```javascript
import * as PIXI from "pixi.js";
import * as PIXIParticles from "pixi-particles";
```

I've no problem using `PIXI.Container` or `PIXI.Particles.Emitter`. But `PIXI.particles.PathParticle` results to be `undifined`. How should be included? is it not part of the `PIXI.particles` namespace

**Comment by andrewstart** [comment]
PathParticle is part of an optional submodule, to make the main library lighter for people that don't need it.
After importing `pixi-particles`, do `require('pixi-particles/modules/path-particle');`, which should make `PIXI.particles.PathParticle` available.

**Comment by edap** [comment]
What if I do not want to use the CommonJS syntax (using `require`) but i want to stick to es6 Syntax?

**Comment by edap** [comment]
Also using the CommonJS syntax is not working:
```javascript
import * as PIXI from "pixi.js";
require("pixi-particles/dist/modules/path-particle");
```

`path-particle.js:63Uncaught TypeError: Cannot read property 'prototype' of undefined(…)`

This happens around line 63
`var s = Particle.prototype;`
where Particle is not defined.

**Comment by andrewstart** [comment]
I've published version 2.0.0 which has everything in one output file.

**Comment by edap** [comment]
thank you ;)
